frontend/src/scenes/session-recordings/apm/NetworkView.tsx

```diff
@@ -0,0 +1,199 @@
+import './NetworkView.scss'
+
+import { LemonTable, Link } from '@posthog/lemon-ui'
+import { BindLogic, useActions, useValues } from 'kea'
+import { CopyToClipboardInline } from 'lib/components/CopyToClipboard'
+import { IconChevronLeft, IconChevronRight } from 'lib/lemon-ui/icons'
+import { LemonButton } from 'lib/lemon-ui/LemonButton'
+import { LemonDivider } from 'lib/lemon-ui/LemonDivider'
+import { LemonSkeleton } from 'lib/lemon-ui/LemonSkeleton'
+import { Tooltip } from 'lib/lemon-ui/Tooltip'
+import AssetProportions from 'scenes/session-recordings/apm/components/AssetProportions'
+import { PerformanceCardRow } from 'scenes/session-recordings/apm/components/PerformanceCard'
+import { MethodTag, StatusTag } from 'scenes/session-recordings/apm/playerInspector/ItemPerformanceEvent'
+import { NetworkBar } from 'scenes/session-recordings/apm/waterfall/NetworkBar'
+
+import { PerformanceEvent } from '~/types'
+
+import { networkViewLogic } from './networkViewLogic'
+
+function SimpleURL({ name, entryType }: { name: string | undefined; entryType: string | undefined }): JSX.Element {
+    // TODO we should show hostname if it isn't the same as the navigation for the page(s) we're looking at
```

Comment:
Not a must for this PR I believe, right?

Reply:
right, it's behind a flag... definitely should before wider release 👍